src/main/webapp/EditProfilePage.html

```diff
@@ -66,5 +84,34 @@ <h1 class="text-center">Edit Your Profile Page</h1>
         </div>
       </form>
     </div>
+
+    <script>
+      (function () {
```

Comment:
I think it would be a better idea to add this function in the user-script.js file instead of writing it in the HTML document.

Reply:
I tried putting it in the user script, but it wouldn't work

Reply:
I think that it is because you are calling the function before the html page is fully loaded. If you wrap your code with this jQuery function it should work:
```
$(document).ready(function() {
});
```

Reply:
Oh okay- I will try that